cedar-lean/Cedar/Thm/Data/List.lean

```diff
@@ -103,8 +103,8 @@ theorem map_equiv (f : α → β) (xs ys : List α) :
   exists p <;>
   rw [List.subset_def] at a b <;>
   simp only [and_true]
-  exact a h
-  exact b h
+  case left  => exact a h
```

Comment:
A very minor suggestion that can be safely ignored:  For these very short proofs about cases,  I prefer to use the `.` notation to separate them out instead of explicitly spelling out the case.  In VSCode, you type `\.` to  generate the bullet point anonymous case marker.  It looks cool :)

```
simp only [and_true]
· exact a h  
· exact b h  
```

Reply:
Looks good to me!  Made a pass doing this.  When the case names are informative (`nil`, `cons`, `some`, `none`, etc) I like having them there, but when they are not (looking at you, `split` and `rcases`) I like this notation when the proofs in each case are short.